projects/ngrx.io/content/guide/component-store/read.md

```diff
@@ -173,6 +173,57 @@ private readonly fetchMoviesData$ = this.select(
 );
 </code-example>
 
+## `selectSignal` method
+
+ComponentStore also provides the `selectSignal` method, which has two signatures. The first signature creates a signal from the provided state projector function, while the second creates a signal by combining provided signals, similar to the select method that combines provided observables.
+
+<code-example header="users.store.ts">
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+
+import { User } from './user.model';
+
+type UsersState = { users: User[]; query: string };
+
+@Injectable()
+export class UsersStore extends ComponentStore&lt;UsersState&gt; {
+  // type: Signal<User[]>
+  readonly users = this.selectSignal((s) => s.users);
+  // type: Signal<string>
+  readonly query = this.selectSignal((s) => s.query);
+  // type: Signal<User[]>
+  readonly filteredUsers = this.selectSignal(
+    this.users,
+    this.query,
+    (users, query) =>
+      users.filter(({ name }) => name.includes(query))
+  );
+}
+</code-example>
+
+Similar to the `computed` function, the `selectSignal` method also accepts the equality function to stop the recomputation of the deeper dependency chain if two values are determined to be equal.
+
+The `state` signal is another addition to the ComponentStore. Instead of using `selectSignal`, it can be used together with the `computed` function to create derived signals.
```

Comment:
let's add another heading before this sentence:

```suggestion
## `state` signal

The `state` signal is another addition to the ComponentStore. Instead of using `selectSignal`, it can be used together with the `computed` function to create derived signals.
```